### ISSUE TITLE
Images and movies should be added as feed enclosures

### ISSUE THREAD

**Issue by ghost** [digitalroastery/weblounge#286]
Currently, the image element can be configured to be added to the feeds. However, this is not the way it's intended to be: images should be added as "enclosures".

**Comment by myniva** [comment]
Isn't this a duplicate of the closed issue #291 ?

**Comment by ghost** [comment]
No. The closed issue is displaying images in the <content> section of the feed. This ticket is talking about media enclosures as they are used in iTunes-like feeds.
